index.html

```diff
@@ -104,10 +104,10 @@ <h2>Contact me</h2>
             a feature that you need built or a project that needs coding.
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224449129-c453cb5e-1396-4dc9-a60f-df1ea83b1619.png)

- [ ] Kindly add ESLint into your linters setup because this linter checks your javascript file errors and those errors can break your whole application 😨 Kindly add this linter to prevent your application from breaking 😊

Reply:
@HAMMAS-SALEEM , hi !
Please, the ESLint check is inside the node_modules for my configuration

Reply:
Hey @touredri
No dear, you will need to put the configuration in your linters.yml file then your linters will be shown here with other linters :blush:

Reply:
Kindly replace your linters.yml file content with this content :arrow_down:

```
name: Linters

on: pull_request

env:
  FORCE_COLOR: 1

jobs:
  lighthouse:
    name: Lighthouse
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: "18.x"
      - name: Setup Lighthouse
        run: npm install -g @lhci/cli@0.7.x
      - name: Lighthouse Report
        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
  webhint:
    name: Webhint
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: "18.x"
      - name: Setup Webhint
        run: |
          npm install --save-dev hint@7.x
          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
      - name: Webhint Report
        run: npx hint .
  stylelint:
    name: Stylelint
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: "18.x"
      - name: Setup Stylelint
        run: |
          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
      - name: Stylelint Report
        run: npx stylelint "**/*.{css,scss}"
  eslint:
    name: ESLint
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: "18.x"
      - name: Setup ESLint
        run: |
          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
      - name: ESLint Report
        run: npx eslint .
  nodechecker:
    name: node_modules checker
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v2
      - name: Check node_modules existence
        run: |
          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```

Reply:
ok let me try

Reply:
@HAMMAS-SALEEM , Invalid Code Review Request does not count into the code reviews limit.